app/scripts/extensions/nav/dropdownMobile.js

```diff
@@ -24,38 +24,38 @@ angular.module('openshiftConsole')
         dropdownItems = dropdownItems.concat([{
           type: 'dom',
           node: [
-            '<li>',
+            '<li class="list-group-item">',
```

Comment:
We've also got a bug that's related.  As it stands, primary nav links need an ng-click="nav.showMobileNav = false;" to correctly close the nav at mobile.